mars/scheduler/graph.py

```diff
@@ -550,6 +550,9 @@ def prepare_graph(self, compose=True):
             for n in tileable_key_opid_to_tiled[(tk, topid)][-1].chunks:
                 self._terminal_chunk_op_tileable[n.op.key].add(tk)
                 self._target_tileable_chunk_ops[tk].add(n.op.key)
+                # treat fetch chunks (from sealed mutable tensor) as executed, see GH#713
```

Comment:
It's better to not add fetch chunks to `self._target_tileable_chunk_ops` rather than add them into finished.